app/components/markdown-block.js

```diff
@@ -1,43 +1,221 @@
+  );
+}
+ 
+  console.log("blocks1", blocks);
+  
+  
+   return (
     <div className={cx(classes.markdownBlock, classes[mode], className)} {...otherProps}>
       <div className={classes.wrapper}>
-        <MarkDown className={classes.mdclasses} options={{ overrides: { ActionButton: { component: ActionButton } } }}>
-          {children}
-        </MarkDown>
+        {subject && <h2 className={classes.subject}>{subject}</h2>}
+        {textSection}
       </div>
     </div>
-  )
-}
-export default TextBlock
+  );
+
+ 
+
+};
+
+export default MarkdownBlock
 
 const useStylesFromThemeFunction = createUseStyles(theme => ({
   markdownBlock: {
-    textAlign: 'center',
-    paddingTop: '1rem',
-    paddingBottom: '4rem',
+     textAlign: 'center',
+     paddingTop: '1rem',
+   
+    
+  },
+  markdownBlockTopImg: {
+     textAlign: 'center',
+    maxWidth: theme.maxPanelWidth,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    // paddingTop: '3rem',
+    paddingBottom: '3rem',
+    display: 'flex',
+    flexDirection: props => (props.imgSide === 'top' ? 'row' : 'row'),
+    // textAlign: 'center',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      flexDirection: (props) => (props.imgSide === 'top' ? 'column' : 'column'),
+    },
+    
   },
   wrapper: {
+     display: 'flex',
+     flexDirection: props => (props.imgSide === 'top' ? 'column' : 'column'),
+    maxWidth: theme.maxPanelWidth,
+    // justifyContent: 'space-between',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    // whiteSpace: 'pre-line',
+  },
+  wrapperWithImage: {
+    display: 'flex',
+    flexDirection: (props) => (props.imgSide === 'top' ? 'row' : 'row'),
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+      flexDirection: (props) => (props.imgSide === 'top' ? 'row' : 'column'),
+      maxWidth: '80%',
+    },
     maxWidth: theme.maxPanelWidth,
     marginLeft: 'auto',
     marginRight: 'auto',
     whiteSpace: 'pre-line',
   },
+  subject: {
+    fontSize: '2rem',
+    fontFamily: 'Montserrat',
+    fontStyle: 'normal',
+    fontWeight: 700,
+    textAlign: 'left',
+    maxWidth: theme.maxPanelWidth,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: 0,
+  },
+  // imageWrapper: {
+  //   display: 'inline-block',
+  //   textAlign: 'center',
+  //   marginBottom: '1rem', 
+  // },
+  imageStyle: {
+    aspectRatio: 2/2,
+    width: props => (props.imgSide === 'top' ? '25%' : '50%'),  
+    height: props => (props.imgSide === 'top' ? '25%' : '80%'),
+    objectFit: props => (props.imgSide === 'top' ? 'cover' : 'unset'),
+    margin: '1rem',
+    borderRadius: '1rem',
+    [`@media (max-width: ${theme.condensedWidthBreakPoint})`]: {
+        width: props => (props.imgSide === 'top' ? '60%' : '80%'),
+        height: props => (props.imgSide === 'top' ? '60%' : '80%'),
+    },
+  },
+  imageStyleTop: {
+    aspectRatio:2/2,
+    width: '14rem',  
```

Comment:
Hardcoding the width like this isn't responsive or reusable. What if there are more or fewer blocks?  What is the screen size is a little bigger than the condensedWidthBreakPoint?